test_contextual_lenses/test_pooling.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for contextual_lenses.py."""
+
+
+import jax.numpy as jnp
+
+import numpy as np
+
+from absl.testing import parameterized
+from absl.testing import absltest
+
+from contextual_lenses import mean_pool, max_pool
+
+
+"""Test cases."""
+test1 = {
+          'testcase_name': 'test1',
```

Comment:
If you're using test cases with uninformative names, then you can avoid specifying testcase_name. Just remove that entry from the dict, and use parameterized.parameters() instead of parameterized.named_parameters below.